data/api/client-server/definitions/key_backup_auth_data.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 The Matrix.org Foundation C.I.C
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+type: object
+title: AuthData
+description: |-
+  The format of the `auth_data` when a key backup is created with the
+  `algorithm` set to `m.megolm_backup.v1.curve25519-aes-sha2`.
+properties:
+  public_key:
+    type: string
+    description: |-
+      The curve25519 public key used to encrypt the backups, encoded in unpadded base64.
+    example: "abcdefg"
```

Comment:
examples in this file were taken from https://github.com/uhoreg/matrix-doc/blob/e2e_backup/proposals/1219-storing-megolm-keys-serverside.md.